nix/pkgs/mayastor/default.nix

```diff
@@ -132,4 +132,38 @@ in
       install * $out/bin
     '';
   };
+
+  # this image to simplify troubleshooting at existing clusters
+  client = stdenv.mkDerivation {
+    name = "mayastor-client";
+    inherit version;
+    src = [
+      ../../../target/debug/mayastor-client
```

Comment:
not sure this is going to work for CI... if it gets scheduled on a workspace were the debug build was not done

Reply:
Thank you for the comment. I hope that this is solved by producing `mayastor-client` as overlay on the top of the `mayastor`.  So, "standard" way current.